apps/frontend/src/components/questionary/questionaryComponents/FapReviewBasis/QuestionaryComponentFapReviewBasis.tsx

```diff
@@ -0,0 +1,210 @@
+          Comment
+        </InputLabel>
+
+        <Editor
+          id="comment"
+          initialValue={state?.fapReview.comment || ''}
+          value={localComment}
+          init={{
+            skin: false,
+            content_css: false,
+            plugins: ['link', 'preview', 'code', 'charmap', 'wordcount'],
+            toolbar: 'bold italic',
+            branding: false,
+            init_instance_callback: (editor) => {
+              handleCharacterCount(editor);
+            },
+          }}
+          onEditorChange={(content, editor) => {
+            const isStartContentDifferentThanCurrent =
+              editor.startContent !== editor.contentDocument.body.innerHTML;
+
+            if (isStartContentDifferentThanCurrent || editor.isDirty()) {
+              handleCharacterCount(editor);
+            }
+            setLocalComment(content);
+          }}
+          onBlur={() =>
+            dispatch({
+              type: 'ITEM_WITH_QUESTIONARY_MODIFIED',
+              itemWithQuestionary: { comment: localComment },
+            })
+          }
+        />
+        <FormHelperText>
+          Characters: {numberOfChars} / {6000}
+        </FormHelperText>
+        <ErrorMessage name={commentFieldId} />
+        <Box marginTop={1} width={150}>
+          <Field
+            name={gradeFieldId}
+            label="Grade"
+            component={gradeDecimalPoints === 1 ? Select : TextField}
+            MenuProps={{ 'data-cy': 'grade-proposal-options' }}
+            onChange={(event: ChangeEvent<HTMLInputElement>) => {
+              dispatch({
+                type: 'ITEM_WITH_QUESTIONARY_MODIFIED',
+                itemWithQuestionary: { grade: +event.target.value },
+              });
+            }}
+            formControl={{
+              fullWidth: true,
+              required: true,
+              margin: 'normal',
+            }}
+            inputProps={
+              gradeDecimalPoints === 1
+                ? {
+                    id: 'grade-proposal',
+                  }
+                : {
+                    id: 'grade-proposal',
+                    step: gradeDecimalPoints,
+                    inputMode: 'decimal',
+                    type: 'number',
+                    min: '1',
+                    max: '10',
+                  }
+            }
+            data-cy="grade-proposal"
+            labelId="grade-proposal-label"
+            options={
+              gradeDecimalPoints === 1
+                ? [...Array(10)].map((e, i) => ({
+                    text: (i + 1).toString(),
+                    value: i + 1,
+                  }))
+                : undefined
+            }
+          />
+        </Box>
+        <NavigationFragment>
+          <ButtonWithDialog label="Grading guide" data-cy="grade-guide">
+            {fap ? <GradeGuidePage fap={fap} /> : <GradeGuidePage />}
+          </ButtonWithDialog>
+          {hasAccessRights && (
+            <Field
```

Comment:
I know I mentioned this in the demo but might be good idea to move this to the end (last step) or add a button of submit instead of checkbox if possible

Reply:
I moved the checkbox to the Summary page.